src/main/java/net/minecraftforge/common/DimensionManager.java

```diff
@@ -132,14 +132,14 @@ public static void init()
 
     public static void registerDimension(int id, DimensionType type)
     {
-        registerDimension(id,type,"noModnameProvided.dimension" + id);
+        registerDimension(id,type,"noModid:dimension" + id);
     }
     
     public static void registerDimension(String id, DimensionType type)
     {
-    	if(id.indexOf(".")==-1)
```

Comment:
Sorry, you should actually use `id.indexOf('.')` because checking a character is the fastest

Reply:
no worries :) but it's index of ":" right because we're using modid:name? Just checking for my sake.

Reply:
Use single quotes